ajax/libs/input-autogrow/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "input-autogrow",
+  "filename": "input-autogrow.min.js",
+  "version": "1.1.0",
+  "description": "Autogrow plugin for inputs",
+  "license": "MIT",
+  "keywords": [
+    "autogrow",
+    "auto-grow",
+    "input",
+    "jquery",
+    "zepto",
+    "javascript",
+    "js"
```

Comment:
Let's add more keywords to let people search this library more easier.
Like `autosize`, `html-forms`.
What do you think?
Many thanks.

![screen shot 2018-05-07 at 12 42 25 am](https://user-images.githubusercontent.com/8349587/39675578-9f470b02-518f-11e8-8b4a-cc5f15ec47de.png)

Reply:
I'd happily accept a PR in `input-autogrow` for more/better keywords.